packages/pages-shared/__tests__/asset-server/handler.test.ts

```diff
@@ -17,11 +17,18 @@ describe("asset-server handler", () => {
 					from: `/${status}`,
 					to: "/home",
 				}))
-				.concat({
-					status: 302,
-					from: "/500",
-					to: "/home",
-				})
+				.concat(
+					{
+						status: 302,
+						from: "/500",
+						to: "/home",
+					},
+					{
+						status: 200,
+						from: "/200",
+						to: "/proxied-file",
```

Comment:
Thanks for the PR! Mind adding a test for what happens when the user proxies to an external URL? Even though wrangler prevents attempting to proxy to an external URL, it's still possible to call the deployment API manually with `to: https://example.com`

Reply:
I'm looking at how to do this - there's not any particularly nice way, open to suggestions. If anything, I'd expect that validation to be done on the side of the deployment API also 😅

Reply:
No worries, I wasn't really sure if there was a good way to test this. But the implementation looks fine so I think we're good.